pulser/simulation/simulation.py

```diff
@@ -419,13 +419,26 @@ def write_samples(
                 noise_amp = np.random.normal(1.0, 1.0e-3) * np.exp(
                     -((r / w0) ** 2)
                 )
-            samples_dict["amp"][slot.ti : slot.tf] = (
+            samples_dict["amp"][slot.ti : slot.tf] += (
                 _pulse.amplitude.samples * noise_amp
             )
-            samples_dict["det"][slot.ti : slot.tf] = (
+            samples_dict["det"][slot.ti : slot.tf] += (
                 _pulse.detuning.samples + noise_det
             )
-            samples_dict["phase"][slot.ti : slot.tf] = _pulse.phase
+            samples_dict["phase"][slot.ti : slot.tf] += _pulse.phase
```

Comment:
It would be nice that these changes were not repeated here. They are already in `develop`, so if you want them to be here the way to go would be to get them from there (whether you merge `develop` to `noisy-xy` and then from `noisy-xy` to here or straight from `develop` will depend on whether we will keep this PR targeting `noisy-xy` or change to `develop`)